doc/releases/reef.rst

```diff
@@ -4,6 +4,28 @@ Reef
 
 Reef is the 18th stable release of Ceph. It is named after the reef squid (Sepioteuthis).
 
+v18.2.6 Reef
+============
+
+This is the sixth release in the Reef series. We recommend that all users update to this release.
```

Comment:
Nit: Should we indicate that this is a hotfix release?

Not pushing a fix because I'm not sure.